wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+### List of obtainable Mod Introduction medals
 
-The **Mod Introduction** medals can be obtained by clearing maps with the specified game modifier. They are meant to encourage newer players to try out each mod and explore what the game has to offer. Do note that if the player plays with multiple game modifiers at a time, the player is **not** eligible to earn *any* of the game modifier medals for that play.
+| Medal name | Requirement(s) |
+| :-- | :-- |
+| Finality | Clear any Qualified, Ranked, or Approved beatmap with the [SuddenDeath](/wiki/Game_modifier/Sudden_Death) mod enabled. |
+| Perfectionist | Clear any Qualified, Ranked, or Approved beatmap with the [Perfect](wiki/Game_modifier/Perfect) mod enabled. |
+| Rock Around The Clock | Clear any Qualified, Ranked, or Approved beatmap with the [HardRock](/wiki/Game_modifier/Hard_Rock) mod enabled. |
+| Time And A Half | Clear any Qualified, Ranked, or Approved beatmap with the [DoubleTime](/wiki/Game_modifier/Double_Time) mod enabled. |
+| Sweet Rave Party | Clear any Qualified, Ranked, or Approved beatmap with the [Nightcore](/wiki/Game_modifier/Nightcore) mod enabled. |
+| Blindsight | Clear any Qualified, Ranked, or Approved beatmap with the [Hidden](/wiki/Game_modifier/Hidden) mod enabled. |
+| Are You Afraid Of The Dark? | Clear any Qualified, Ranked, or Approved beatmap with the [Flashlight](/wiki/Game_modifier/Flashlight) mod enabled. |
+| Dial It Right Back | Clear any Qualified, Ranked, or Approved beatmap with the [Easy](/wiki/Game_modifier/Easy) mod enabled. |
+| Risk Averse | Clear any Qualified, Ranked, or Approved beatmap with the [NoFail](/wiki/Game_modifier/No_Fail) mod enabled. |
+| Slowboat | Clear any Qualified, Ranked, or Approved beatmap with the [HalfTime](/wiki/Game_modifier/Half_Time) mod enabled. |
+| Burned Out | Clear any Qualified, Ranked, or Approved beatmap with the [SpunOut](/wiki/Game_modifier/Spun_Out) mod enabled. This medal can only be achieved in the osu! game mode as the SpunOut mod is not present in other game modes. |
 
 ## Hush-Hush
 
-Medals of this kind have no obvious way to get them. The player must find out what needs to be done in order to unlock the medal. The title itself is a clue.
+The **Beatmap Packs medals** are "secret" medals that are being awarded upon pulling up certain specific feats, such as clearing a particular beatmap under some peculiar circumstances or setting down a score with an unusual mod combination. Although at first it may appear that there are no clear-cut instructions as to how the Hush-Hush medals could be unlocked, there are some hints that are being put around to guide all the curious medal hunters in their sleuth as follows:
+
+- The name of the medals themselves,
+- The medals' brief description, and
+- The *flair text* <!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> of the medals (if any), viewable by hovering over the corresponding medals' card.
```

Comment:
```suggestion
*Beatmap Packs medals* are "secret" medals that are being awarded upon pulling up certain specific feats, such as clearing a particular beatmap under some peculiar circumstances or setting down a score with an unusual mod combination. Although it may appear as if there are no clear-cut instructions as to how the Hush-Hush medals could be unlocked, there are some hints that are being put around to guide all the curious medal hunters in their sleuth as follows:

- The name of the medals themselves,
- The medals' brief description, and
- The *flair text* <!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> of the medals (if any), viewable by hovering over the corresponding medals' card.
```
this isnt bad but i would *personally* write it a bit differently, something like this:
```suggestion
*Beatmap Packs medals* are medals that are awarded for completing very specific feats, which are not told to the player (hence the name). These feats can range anywhere from completing a beatmap under peculiar circumstances to setting a score with an unusual mod combination. Despite their secretive nature, however, some hints are given as to how to obtain the medal, such as the name of the medal itself, the medal's flair text, and the medal's description. <!-- https://twitter.com/ephemeralis/status/1068054191602638850 -->
```

Reply:
applied this in a way that blends in to the sentence that i'm having right now, tyty